docs/data/material/guides/nextjs/nextjs.md

```diff
@@ -98,11 +98,11 @@ Then in `src/app/layout.tsx`, pass the theme to `ThemeProvider`:
    return (
      <html lang="en">
        <body>
-          <AppRouterCacheProvider>
-+           <ThemeProvider theme={theme}>
+-         <AppRouterCacheProvider>
```

Comment:
This doesn't look right, `AppRouterCacheProvider` should still be needed 🤔 @shqear93